quadro/quadro_tests.cpp

```diff
@@ -14,8 +14,25 @@ bool is_equal(const double d1, const double d2) {
     else return 0; 
 }
 
+bool is_roots_equal(roots res_roots, roots exp_roots) {
```

Comment:
wtf...

Reply:
rename to `is_equal`. it is obvious that it compares roots structs. this information is in the declaration of this function

Reply:
resolved